app/frontend/src/components/FileStatus/DocumentsDetailList.tsx

```diff
@@ -57,6 +61,30 @@ export const DocumentsDetailList = ({ items, onFilesSorted}: Props) => {
         alert(`Item invoked: ${item.name}`);
     }
 
+    const [itemList, setItems] = useState<IDocument[]>(items);
+    function retryErroredFile(item: IDocument): void {
+        retryFile(item.filePath)
+            .then(() => {
+                // Create a new array with the updated item
+                const updatedItems = itemList.map((i) => {
+                    if (i.key === item.key) {
```

Comment:
Updating the status of the item in the UX is not desired as ultimately it needs to be retrieved from the CosmosDB. Besides the status for a file uploaded from the UX is "Uploaded "and not "Queued" to start with. It would be better to simply trigger a reload of all the states here.

Reply:
@dayland  you can't reload all of the status because there is a delay from when the blob is re-triggered to cosmos being updated.  The status is updated after about ~5 seconds after the click which is why @ryonsteele  and I took the UI changes this direction.